nemo_skills/evaluation/evaluator/ioi.py

```diff
@@ -88,76 +89,60 @@ def _precompile_grader(
         wait_for_sandbox(sandbox)
         sandbox._owner_tid = threading.get_ident()
 
-    pre_dir = f"/tmp/ioi_pre_{problem_name}_{os.getpid()}"
-    # Build shell script to create files and invoke compile.sh.
-    creation_cmds = [
-        f"mkdir -p {pre_dir}/graders",
-    ]
-    # Dump grader related files
+    pre_dir = f"/nemo_run/ioi_pre_{problem_name}_{os.getpid()}"
```

Comment:
What's the reason for writing to the `/nemo_run` dir? Rather than creating a new mount like `/workspace/ioi_problems/`? I realize this `/nemo_run/ioi_pre_{problem_name}_{os.getpid()}` is not a directory that likely is made by `nemo-run` at any point--but I get nervous writing to `/nemo_run` as it is kind of supposed to be a static snapshot of what was run and this would be modifying that directory at runtime.